src/app/pages/research/research.scss

```diff
@@ -37,6 +37,12 @@
         margin-block: 0 !important;
     }
 
+    // Changes
+    // Adding changes from PM ticker - Mikayla
+    h1{
+        color: #151b2c;
```

Comment:
Instead of making the change here like this (this will affect all <h1> tags), i would make the change here for the specific h1

https://github.com/openstax/os-webview/blob/main/src/app/pages/research/components/header.js#L24